Comparing source compatibility of /Users/oovever/Documents/projcet/Github/netty/transport-sctp/target/netty-transport-sctp-4.1.80.Final-SNAPSHOT.jar against /Users/oovever/.m2/repository/io/netty/netty-transport-sctp/4.1.79.Final/netty-transport-sctp-4.1.79.Final.jar
WARNING: You have ignored certain classes, i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC io.netty.channel.sctp.DefaultSctpChannelConfig  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelConfig
	===  UNCHANGED INTERFACE: io.netty.channel.sctp.SctpChannelConfig
	===  UNCHANGED SUPERCLASS: io.netty.channel.DefaultChannelConfig (<- io.netty.channel.DefaultChannelConfig)
	===  UNCHANGED CONSTRUCTOR: PUBLIC DefaultSctpChannelConfig(io.netty.channel.sctp.SctpChannel, com.sun.nio.sctp.SctpChannel)
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams getInitMaxStreams()
	===  UNCHANGED METHOD: PUBLIC java.lang.Object getOption(io.netty.channel.ChannelOption)
	===  UNCHANGED METHOD: PUBLIC java.util.Map getOptions()
	===  UNCHANGED METHOD: PUBLIC int getReceiveBufferSize()
	===  UNCHANGED METHOD: PUBLIC int getSendBufferSize()
	===  UNCHANGED METHOD: PUBLIC boolean isSctpNoDelay()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setAllocator(io.netty.buffer.ByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setAutoClose(boolean)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setAutoRead(boolean)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setConnectTimeoutMillis(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setInitMaxStreams(com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setMaxMessagesPerRead(int)
		===  UNCHANGED ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setMessageSizeEstimator(io.netty.channel.MessageSizeEstimator)
	===  UNCHANGED METHOD: PUBLIC boolean setOption(io.netty.channel.ChannelOption, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setReceiveBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setRecvByteBufAllocator(io.netty.channel.RecvByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setSctpNoDelay(boolean)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setSendBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setWriteBufferHighWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setWriteBufferLowWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setWriteBufferWaterMark(io.netty.channel.WriteBufferWaterMark)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig setWriteSpinCount(int)
===  UNCHANGED CLASS: PUBLIC io.netty.channel.sctp.DefaultSctpServerChannelConfig  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelConfig
	===  UNCHANGED INTERFACE: io.netty.channel.sctp.SctpServerChannelConfig
	===  UNCHANGED SUPERCLASS: io.netty.channel.DefaultChannelConfig (<- io.netty.channel.DefaultChannelConfig)
	===  UNCHANGED CONSTRUCTOR: PUBLIC DefaultSctpServerChannelConfig(io.netty.channel.sctp.SctpServerChannel, com.sun.nio.sctp.SctpServerChannel)
	===  UNCHANGED METHOD: PUBLIC int getBacklog()
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams getInitMaxStreams()
	===  UNCHANGED METHOD: PUBLIC java.lang.Object getOption(io.netty.channel.ChannelOption)
	===  UNCHANGED METHOD: PUBLIC java.util.Map getOptions()
	===  UNCHANGED METHOD: PUBLIC int getReceiveBufferSize()
	===  UNCHANGED METHOD: PUBLIC int getSendBufferSize()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setAllocator(io.netty.buffer.ByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setAutoClose(boolean)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setAutoRead(boolean)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setBacklog(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setConnectTimeoutMillis(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setInitMaxStreams(com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setMaxMessagesPerRead(int)
		===  UNCHANGED ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setMessageSizeEstimator(io.netty.channel.MessageSizeEstimator)
	===  UNCHANGED METHOD: PUBLIC boolean setOption(io.netty.channel.ChannelOption, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setReceiveBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setRecvByteBufAllocator(io.netty.channel.RecvByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setSendBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setWriteBufferHighWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setWriteBufferLowWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setWriteBufferWaterMark(io.netty.channel.WriteBufferWaterMark)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig setWriteSpinCount(int)
===  UNCHANGED CLASS: PUBLIC io.netty.channel.sctp.nio.NioSctpChannel  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.Channel
	===  UNCHANGED INTERFACE: io.netty.util.AttributeMap
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundInvoker
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: io.netty.channel.sctp.SctpChannel
	===  UNCHANGED SUPERCLASS: io.netty.channel.nio.AbstractNioMessageChannel (<- io.netty.channel.nio.AbstractNioMessageChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC NioSctpChannel(com.sun.nio.sctp.SctpChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC NioSctpChannel()
	===  UNCHANGED CONSTRUCTOR: PUBLIC NioSctpChannel(io.netty.channel.Channel, com.sun.nio.sctp.SctpChannel)
	===  UNCHANGED METHOD: PUBLIC java.util.Set allLocalAddresses()
	===  UNCHANGED METHOD: PUBLIC java.util.Set allRemoteAddresses()
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.Association association()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig config()
	===  UNCHANGED METHOD: PROTECTED void doBind(java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doClose()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED boolean doConnect(java.net.SocketAddress, java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doDisconnect()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doFinishConnect()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED int doReadMessages(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED boolean doWriteMessage(java.lang.Object, io.netty.channel.ChannelOutboundBuffer)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED FINAL java.lang.Object filterOutboundMessage(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC boolean isActive()
	===  UNCHANGED METHOD: PROTECTED com.sun.nio.sctp.SctpChannel javaChannel()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress localAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress localAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelMetadata metadata()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannel parent()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress remoteAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress remoteAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
===  UNCHANGED CLASS: PUBLIC io.netty.channel.sctp.nio.NioSctpServerChannel  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.Channel
	===  UNCHANGED INTERFACE: io.netty.util.AttributeMap
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundInvoker
	===  UNCHANGED INTERFACE: io.netty.channel.sctp.SctpServerChannel
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: io.netty.channel.ServerChannel
	===  UNCHANGED SUPERCLASS: io.netty.channel.nio.AbstractNioMessageChannel (<- io.netty.channel.nio.AbstractNioMessageChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC NioSctpServerChannel()
	===  UNCHANGED METHOD: PUBLIC java.util.Set allLocalAddresses()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig config()
	===  UNCHANGED METHOD: PROTECTED void doBind(java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doClose()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED boolean doConnect(java.net.SocketAddress, java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doDisconnect()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doFinishConnect()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED int doReadMessages(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED boolean doWriteMessage(java.lang.Object, io.netty.channel.ChannelOutboundBuffer)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.Object filterOutboundMessage(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC boolean isActive()
	===  UNCHANGED METHOD: PROTECTED com.sun.nio.sctp.SctpServerChannel javaChannel()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress localAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress localAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelMetadata metadata()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress remoteAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress remoteAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
===  UNCHANGED CLASS: PUBLIC io.netty.channel.sctp.oio.OioSctpChannel  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.Channel
	===  UNCHANGED INTERFACE: io.netty.util.AttributeMap
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundInvoker
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: io.netty.channel.sctp.SctpChannel
	===  UNCHANGED SUPERCLASS: io.netty.channel.oio.AbstractOioMessageChannel (<- io.netty.channel.oio.AbstractOioMessageChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC OioSctpChannel(io.netty.channel.Channel, com.sun.nio.sctp.SctpChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC OioSctpChannel(com.sun.nio.sctp.SctpChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC OioSctpChannel()
	===  UNCHANGED METHOD: PUBLIC java.util.Set allLocalAddresses()
	===  UNCHANGED METHOD: PUBLIC java.util.Set allRemoteAddresses()
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.Association association()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpChannelConfig config()
	===  UNCHANGED METHOD: PROTECTED void doBind(java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doClose()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doConnect(java.net.SocketAddress, java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doDisconnect()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED int doReadMessages(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doWrite(io.netty.channel.ChannelOutboundBuffer)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.Object filterOutboundMessage(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC boolean isActive()
	===  UNCHANGED METHOD: PUBLIC boolean isOpen()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress localAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress localAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelMetadata metadata()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannel parent()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress remoteAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress remoteAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED CLASS: PUBLIC io.netty.channel.sctp.oio.OioSctpServerChannel  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.Channel
	===  UNCHANGED INTERFACE: io.netty.util.AttributeMap
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundInvoker
	===  UNCHANGED INTERFACE: io.netty.channel.sctp.SctpServerChannel
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: io.netty.channel.ServerChannel
	===  UNCHANGED SUPERCLASS: io.netty.channel.oio.AbstractOioMessageChannel (<- io.netty.channel.oio.AbstractOioMessageChannel)
	===  UNCHANGED CONSTRUCTOR: PUBLIC OioSctpServerChannel()
	===  UNCHANGED CONSTRUCTOR: PUBLIC OioSctpServerChannel(com.sun.nio.sctp.SctpServerChannel)
	===  UNCHANGED METHOD: PUBLIC java.util.Set allLocalAddresses()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpServerChannelConfig config()
	===  UNCHANGED METHOD: PROTECTED void doBind(java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doClose()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doConnect(java.net.SocketAddress, java.net.SocketAddress)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doDisconnect()
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED int doReadMessages(java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED void doWrite(io.netty.channel.ChannelOutboundBuffer)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED java.lang.Object filterOutboundMessage(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC boolean isActive()
	===  UNCHANGED METHOD: PUBLIC boolean isOpen()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress localAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress localAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelMetadata metadata()
	===  UNCHANGED METHOD: PUBLIC java.net.InetSocketAddress remoteAddress()
	===  UNCHANGED METHOD: PROTECTED java.net.SocketAddress remoteAddress0()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannel  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.Channel
	===  UNCHANGED INTERFACE: io.netty.util.AttributeMap
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundInvoker
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Set allLocalAddresses()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Set allRemoteAddresses()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT com.sun.nio.sctp.Association association()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig config()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.net.InetSocketAddress localAddress()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannel parent()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.net.InetSocketAddress remoteAddress()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelConfig
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams getInitMaxStreams()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getReceiveBufferSize()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getSendBufferSize()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isSctpNoDelay()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setAllocator(io.netty.buffer.ByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setAutoClose(boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setAutoRead(boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setConnectTimeoutMillis(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setInitMaxStreams(com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setMaxMessagesPerRead(int)
		===  UNCHANGED ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setMessageSizeEstimator(io.netty.channel.MessageSizeEstimator)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setReceiveBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setRecvByteBufAllocator(io.netty.channel.RecvByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setSctpNoDelay(boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setSendBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setWriteBufferHighWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setWriteBufferLowWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setWriteBufferWaterMark(io.netty.channel.WriteBufferWaterMark)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpChannelConfig setWriteSpinCount(int)
===  UNCHANGED CLASS: PUBLIC FINAL io.netty.channel.sctp.SctpChannelOption  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.util.Constant
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED SUPERCLASS: io.netty.channel.ChannelOption (<- io.netty.channel.ChannelOption)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_EXPLICIT_COMPLETE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_FRAGMENT_INTERLEAVE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_INIT_MAXSTREAMS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_SET_PEER_PRIMARY_ADDR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_DISABLE_FRAGMENTS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_NODELAY
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.channel.ChannelOption SCTP_PRIMARY_ADDR
===  UNCHANGED CLASS: PUBLIC FINAL io.netty.channel.sctp.SctpMessage  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.util.ReferenceCounted
	===  UNCHANGED INTERFACE: io.netty.buffer.ByteBufHolder
	===  UNCHANGED SUPERCLASS: io.netty.buffer.DefaultByteBufHolder (<- io.netty.buffer.DefaultByteBufHolder)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpMessage(int, int, io.netty.buffer.ByteBuf)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpMessage(int, int, boolean, io.netty.buffer.ByteBuf)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpMessage(com.sun.nio.sctp.MessageInfo, io.netty.buffer.ByteBuf)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage copy()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage duplicate()
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC boolean isComplete()
	===  UNCHANGED METHOD: PUBLIC boolean isUnordered()
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.MessageInfo messageInfo()
	===  UNCHANGED METHOD: PUBLIC int protocolIdentifier()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage replace(io.netty.buffer.ByteBuf)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage retain()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage retain(int)
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage retainedDuplicate()
	===  UNCHANGED METHOD: PUBLIC int streamIdentifier()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage touch()
	===  UNCHANGED METHOD: PUBLIC io.netty.channel.sctp.SctpMessage touch(java.lang.Object)
===  UNCHANGED CLASS: PUBLIC FINAL io.netty.channel.sctp.SctpNotificationHandler  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: com.sun.nio.sctp.NotificationHandler
	===  UNCHANGED SUPERCLASS: com.sun.nio.sctp.AbstractNotificationHandler (<- com.sun.nio.sctp.AbstractNotificationHandler)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpNotificationHandler(io.netty.channel.sctp.SctpChannel)
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.HandlerResult handleNotification(com.sun.nio.sctp.AssociationChangeNotification, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.HandlerResult handleNotification(com.sun.nio.sctp.PeerAddressChangeNotification, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.HandlerResult handleNotification(com.sun.nio.sctp.SendFailedNotification, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC com.sun.nio.sctp.HandlerResult handleNotification(com.sun.nio.sctp.ShutdownNotification, java.lang.Object)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannel  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.Channel
	===  UNCHANGED INTERFACE: io.netty.util.AttributeMap
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundInvoker
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: io.netty.channel.ServerChannel
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Set allLocalAddresses()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture bindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig config()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.net.InetSocketAddress localAddress()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.ChannelFuture unbindAddress(java.net.InetAddress, io.netty.channel.ChannelPromise)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelConfig
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getBacklog()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams getInitMaxStreams()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getReceiveBufferSize()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getSendBufferSize()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setAllocator(io.netty.buffer.ByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setAutoClose(boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setAutoRead(boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setBacklog(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setConnectTimeoutMillis(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setInitMaxStreams(com.sun.nio.sctp.SctpStandardSocketOptions$InitMaxStreams)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setMaxMessagesPerRead(int)
		===  UNCHANGED ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setMessageSizeEstimator(io.netty.channel.MessageSizeEstimator)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setReceiveBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setRecvByteBufAllocator(io.netty.channel.RecvByteBufAllocator)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setSendBufferSize(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setWriteBufferHighWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setWriteBufferLowWaterMark(int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setWriteBufferWaterMark(io.netty.channel.WriteBufferWaterMark)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT io.netty.channel.sctp.SctpServerChannelConfig setWriteSpinCount(int)
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.sctp.SctpInboundByteStreamHandler  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelInboundHandler
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelHandler
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.MessageToMessageDecoder (<- io.netty.handler.codec.MessageToMessageDecoder)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpInboundByteStreamHandler(int, int)
	===  UNCHANGED METHOD: PUBLIC FINAL boolean acceptInboundMessage(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PROTECTED boolean acceptInboundMessage(io.netty.channel.sctp.SctpMessage)
	===  UNCHANGED METHOD: PROTECTED void decode(io.netty.channel.ChannelHandlerContext, io.netty.channel.sctp.SctpMessage, java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.sctp.SctpMessageCompletionHandler  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelInboundHandler
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelHandler
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.MessageToMessageDecoder (<- io.netty.handler.codec.MessageToMessageDecoder)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpMessageCompletionHandler()
	===  UNCHANGED METHOD: PROTECTED void decode(io.netty.channel.ChannelHandlerContext, io.netty.channel.sctp.SctpMessage, java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC void handlerRemoved(io.netty.channel.ChannelHandlerContext)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED CLASS: PUBLIC ABSTRACT io.netty.handler.codec.sctp.SctpMessageToMessageDecoder  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelInboundHandler
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelHandler
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.MessageToMessageDecoder (<- io.netty.handler.codec.MessageToMessageDecoder)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpMessageToMessageDecoder()
	===  UNCHANGED METHOD: PUBLIC boolean acceptInboundMessage(java.lang.Object)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.sctp.SctpOutboundByteStreamHandler  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelOutboundHandler
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelHandler
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.MessageToMessageEncoder (<- io.netty.handler.codec.MessageToMessageEncoder)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpOutboundByteStreamHandler(int, int)
	===  UNCHANGED CONSTRUCTOR: PUBLIC SctpOutboundByteStreamHandler(int, int, boolean)
	===  UNCHANGED METHOD: PROTECTED void encode(io.netty.channel.ChannelHandlerContext, io.netty.buffer.ByteBuf, java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
